projects/plugins/jetpack/modules/search/class-jetpack-search-customize.php

```diff
@@ -126,7 +126,7 @@ public function customize_register( $wp_customize ) {
 		$wp_customize->add_setting(
 			$id,
 			array(
-				'default'   => 'results',
+				'default'   => Jetpack_Search_Options::OVERLAY_TRIGGER_IMMEDIATE,
```

Comment:
If we're changing the default value here, it's not necessary to manually update the value in `auto_config_search`.

We use `auto_config_search` as a way to define an updated `default` value for new customers since directly changing the default value will affect both old and new customers alike.

When we rolled out the new expanded result format, we wanted to preserve our existing customers' `minimal` result format while ensuring all new customers would default to the `expanded` result format.

Reply:
👍 Sounds sensible - I'll remove it. How will the default work in the new customization experience?

Reply:
With this code, any customers that haven't modified their overlay trigger will now use the `immediate` overlay trigger. This will affect both new and existing customers.